### PR TITLE
plugin test and lint should be part of cli e2e test

### LINKED ISSUE #312: plugin test and lint should be part of cli e2e test
e2e test script: https://github.com/spotify/backstage/blob/master/scripts/cli-e2e-test.js

we want to make sure that lint and test commands run successfully in the newly scaffolded plugin

### PR DESCRIPTION
**Pull request by mateusmarquezini**
## Hey, I just made a Pull Request!

Fixes #312 

- Added a new step to run (**yarn lint** and **yarn test**) on CI after plugin creation;
- Fixed some minor **eslint** errors in some code files;

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] [CODEOWNERS](./CODEOWNERS) updated (for new stuff)
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -40,3 +40,9 @@ jobs:
         run: |
           sudo sysctl fs.inotify.max_user_watches=524288
           node scripts/cli-e2e-test.js
+      - name: yarn lint, test after plugin creation
+        run: |
+          yarn lint
+          yarn test
```
Comment (Rugvip) [inline .github/workflows/frontend.yml]: This'll test and lint the entire project, which isn't really what we want, that's taken care of by in a separate task. I think we can solve it by telling GitHub workflows to run the commands inside the plugin folder only.

**Review by Rugvip — APPROVED** [review]
Looks good given the comment is taken care of :grin:

Thanks for taking care of the lint issues! 😅

File: .github/workflows/frontend.yml (hunk 1 above)
Comment (mateusmarquezini) [reply .github/workflows/frontend.yml]: Ok, I thought it needed to consider the new plugin created, not just them (plugins folders). I can continue on this and make this change, no problem.

**Review by Rugvip — APPROVED** [review]
Looks good! Feel free to merge 👍

File: .github/workflows/frontend.yml (hunk 1 above)
Comment (Rugvip) [reply .github/workflows/frontend.yml]: Yeah only running tests inside `plugins/test-plugin` was the idea

**Pull request merged**